packages/builders/scripts/vats/terminate-original-stkATOM-USD-price-feed-governor.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file Terminate v110 zcf-b1-9f877-stkATOM-USD_price_feed-governor.
+ */
+
+import { E } from '@endo/far';
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+/**
+ * @param {BootstrapPowers} powers
+ */
+export const terminateOriginalStkATOMPriceFeedGovernor = async ({
+  consume: { board, governedContractKits },
+}) => {
+  const { Fail, quote: q } = assert;
+  const expectedLabel = 'stkATOM-USD_price_feed';
```

Comment:
I think it's full name is longer. Even if we don't have to match the zcf bundleId, at least include **-governor** to distinguish it from the governed priceFeed.

```suggestion
  const expectedLabel = 'zcf-b1-9f877-stkATOM-USD_price_feed-governor';
```

Reply:
"stkATOM-USD_price_feed" is the value of the `label` field in the kit retrieved from `governedContractKits`, although the governed instance is indeed named "zcf-b1-9f877-stkATOM-USD_price_feed-governor". I thought using `label` rather than `name` would avoid this confusion, but perhaps you have a better idea?

Reply:
Oh, right. You're getting the contractKit, which uniquely has that name. the governor adminFacet there is the one we want.  Your code is correct and I was thinking of the something else.